src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/IndexStateManagementIndices.kt

```diff
@@ -118,5 +119,8 @@ class IndexStateManagementIndices(
         const val HISTORY_WRITE_INDEX_ALIAS = "$HISTORY_INDEX_BASE-write"
         const val HISTORY_INDEX_PATTERN = "<$HISTORY_INDEX_BASE-{now/d{yyyy.MM.dd}}-1>"
         const val HISTORY_ALL = "$HISTORY_INDEX_BASE*"
+
+        fun indexStateManagementMappings() = javaClass.classLoader.getResource("mappings/opendistro-ism-config.json").readText()
```

Comment:
Is this loading the mappings now every time this function is called?
Seems like before it was done just once on the class initialization and now it'll be loading from disk every time we need to compare mappings? Or does getResources cache/memoize anything internally?

Reply:
Changed to `val`